vividus-tests/src/main/resources/properties/environment/integration/environment.properties

```diff
@@ -1,4 +1,4 @@
-web-application.main-page-url=http://example.org/
+web-application.main-page-url=https://vividus-test-site.herokuapp.com
```

Comment:
we need to reuse those https://github.com/vividus-framework/vividus/blob/11d65010c4c6a4499d7b89e5e22cf709a69f9a08/vividus-tests/src/main/resources/properties/environment/environment.properties#L1-L4